webpack/components/extensions/HostDetails/Tabs/ErrataTab/ErrataTab.js

```diff
@@ -120,6 +120,18 @@ export const ErrataTab = () => {
     initialSortColumnName: 'Errata',
   });
 
+  const fetchItemsQuery = () => {
+    let query = '';
```

Comment:
```suggestion
    let query = [];
```
Doing this with an array (and then `return query.join(' & ')`) will be a bit cleaner and easier to read.